server/util/log/log.go

```diff
@@ -79,27 +79,9 @@ func LogHTTPRequest(ctx context.Context, url string, dur time.Duration, statusCo
 }
 
 func init() {
-	// Start us in a "nice" configuration, in case logging
+	// Start us in a "nice" configuration, in case any logging
 	// is done before Configure is called.
-	Configure("info", false)
-}
-
-func Configure(level string, enableStructured bool) error {
-	if enableStructured {
-		log.Logger = StructuredLogger()
-	} else {
-		log.Logger = LocalLogger(level)
-	}
-	intLogLevel := zerolog.InfoLevel
-	if level != "" {
-		if l, err := zerolog.ParseLevel(level); err == nil {
-			intLogLevel = l
-		} else {
-			return err
-		}
-	}
-	zerolog.SetGlobalLevel(intLogLevel)
-	return nil
+	Configure("info", false, false)
```

Comment:
Can we enable this when running tests? 
Perhaps we can check for one of the standard env variables like TEST_TARGET?

Reply:
Maybe we can have a common class like testenv reconfigure the logger to use DEBUG + filename logging?